controllers/articles-controller.js

```diff
@@ -21,5 +21,16 @@ const getArticles = (req, res, next) => {
     });
 };
 
+const patchArticle = (req, res, next) => {
+  const {article_id} = req.params
+  const inc_votes = req.body
+  Promise.all([checkArticleExists(article_id) ,updateArticleVotes(article_id, inc_votes)]).then(([undefined, article]) => {
```

Comment:
we dont need  to check article exists if the query returns row.length ===0 we know that it didnt exist!